webpack.config.js

```diff
@@ -7,7 +7,7 @@ const branch = require('child_process')
   .execSync('git rev-parse --abbrev-ref HEAD', { encoding: 'utf-8' })
   .trim();
 
-const suffix = branch === 'main' ? '' : `-${branch}`;
+const suffix = (branch === 'main' || branch === 'HEAD') ? '' : `-${branch}`;
```

Comment:
This looks like it's for a different ticket and can be reverted?

Reply:
Done